src/main/java/org/generationcp/middleware/service/api/dataset/ObservationUnitsSearchDTO.java

```diff
@@ -41,16 +42,19 @@
 		private Map<String, List<String>> filteredValues;
 		private Map<String, String> filteredTextValues;
 		private Map<String, String> variableTypeMap;
+		private List<String> filterColumns;
 		private Integer variableId;
 
 		public Filter() {
 			this.byMissing = false;
 			this.byOutOfBound = false;
-			this.byOutOfSync =  false;
+			this.byOutOfSync = false;
 			this.byOverwritten = false;
 			this.filteredValues = new HashMap<>();
 			this.filteredTextValues = new HashMap<>();
 			this.variableId = null;
+			/** This is used by Visualization tool, to specify the columns that will included in the data returned from the server. **/
```

Comment:
It's a bit confusing to put `filterColumns` here because the others are filters for **rows** that will be included.
Maybe we should move it as variable of the main DTO class.